library/2.1.5/validations.py

```diff
@@ -135,6 +135,13 @@ def valid_fs_path_or_raise(path: str):
     return path
 
 
+def allowed_fs_host_path_or_raise(path: str):
+    disallowed_paths = ["/mnt", "/audit", "/var/db", "/data"]
+    if path.rstrip("/") in disallowed_paths:
```

Comment:
We have too many disallowed paths. It'll be easier going about it from the other direction. i.e.
```
if not path.startswith('/mnt') or path.rstrip('/') == '/mnt':
    raise error